xbmc/games/dialogs/GUIDialogSelectGameClient.cpp

```diff
@@ -70,6 +70,8 @@ std::string CGUIDialogSelectGameClient::ShowAndGetGameClient(const std::string &
     {
       CFileItemPtr item(XFILE::CAddonsDirectory::FileItemFromAddon(candidate, candidate->ID()));
       item->SetLabel2(g_localizeStrings.Get(35257)); // "Installed"
+      if (item->GetPath() == selected)
```

Comment:
This if reads so weird, it might just be worth to renamed "selected" to something more helpful, like "hasSavestate"

Reply:
Renamed